manifests/base/kustomization.yaml

```diff
@@ -193,3 +193,180 @@ vars:
       fieldpath: data.odh-rstudio-gpu-notebook-n-1
 configurations:
   - params.yaml
+
+configMapGenerator:
+  - name: notebook
+    env: commit.env
+generatorOptions:
+  disableNameSuffixHash: true
+vars:
```

Comment:
We have another `vars` field in this yaml,
for readability and consistency, we can include these in there, line 25